include/oc_rep.h

```diff
@@ -435,6 +435,18 @@ CborError oc_rep_encoder_close_container(CborEncoder *encoder,
     }                                                                          \
   } while (0)
 
+/** Alternative to oc_rep_set_text_string in case we know the length of the
+value */
+#define oc_rep_set_text_string_v1(object, key, value, value_len)               \
```

Comment:
pls in `oc_rep_set_text_string` call `oc_rep_set_text_string_v1`

Reply:
wanted to avoid doing a nil-check twice